ros2_batch_job/__main__.py

```diff
@@ -356,6 +371,51 @@ def build_and_test(args, job):
     # return 0 if ret_test == 0 and ret_testr == 0 else 1
     return 0
 
+def build_with_sanitizer(cmd, args, job):
```

Comment:
Could we add some doc to this method? What kind of shell command do we usually expect from this?

Reply:
Done